tests/hikari/impl/test_cache.py

```diff
@@ -129,18 +129,6 @@ def test_set_dm_channel_id(self, cache_impl):
 
         assert cache_impl._dm_channel_entries == {43123123: 12222}
 
-    def test_set_dm_channel_id_when_user_not_known(self, cache_impl):
-        cache_impl.set_dm_channel_id(StubModel(43123123), StubModel(12222))
-
-        assert cache_impl._dm_channel_entries == {}
-
-    def test_set_dm_channel_id_when_not_enabled(self, cache_impl):
-        cache_impl._user_entries = collections.FreezableDict({43123123: object()})
-        cache_impl._settings.components = 0
-        cache_impl.set_dm_channel_id(StubModel(43123123), StubModel(12222))
```

Comment:
presumably this test case still exists

Reply:
It has been moved down with all the other tests cases for disabled components